### ISSUE TITLE
Implement ObjectWithURL renderer

### ISSUE THREAD

**Issue by linclark** [firefox-devtools/original-console-fork#41]
Example command: `window`

```
<div xmlns="http://www.w3.org/1999/xhtml" id="console-msg-3" class="message cm-s-mozilla" category="output" severity="log" filter="null">
  <span class="timestamp devtools-monospace">14:32:52.551 </span>
  <span class="indent" style="width: 0px;"></span>
  <span class="icon" title="Log"></span>
  <span class="message-body-wrapper message-body devtools-monospace">
    <span>
      <span class="kind-ObjectWithURL">
        <a class="cm-variable" draggable="false" href="#">Window</a> → <a class="undefined" draggable="false" href="about:home">about:home</a>
      </span>
    </span>
  </span>
</div>
```

**Comment by linclark** [comment]
This might end up being handled by reps. See #43

**Comment by linclark** [comment]
This will be fixed by #94
